app/views/candidate_interface/gcse_details/review.html.erb

```diff
@@ -3,7 +3,7 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-     <h1 class="govuk-heading-xl"><%= heading_for_gcse_show(@application_qualification.subject) %></h1>
+     <h1 class="govuk-heading-xl"><%= heading_for_gcse_show(@subject) %></h1>
 
       <section class="app-summary-card govuk-!-margin-bottom-6">
```

Comment:
For summary cards we use Rails components, see `app/components/degrees_review_component.rb` and `app/components/degrees_review_component.html.erb` for example.

The benefit of this is that it can be easily reused when a candidate can reviews their whole application before submitting.

Reply:
Also, the summary card should be full-width instead of two-third. So outside of the `<div class="govuk-grid-column-two-thirds">`.